src/de/value.rs

```diff
@@ -111,27 +111,38 @@ impl de::VariantVisitor for VariantVisitor
         Ok(())
     }
 
-    /// `visit_seq` is called when deserializing a tuple-like variant.
-    fn visit_seq<V>(&mut self, _visitor: V) -> Result<V::Value, Self::Error>
-        where V: de::Visitor
+    fn visit_newtype<T>(&mut self) -> Result<T, Self::Error>
+        where T: de::Deserialize,
     {
-        unimplemented!()
+        if let Some(element) = self.0.take() {
+            if let Content::Text(content) = element.members {
```

Comment:
I'm not really sure about this... `element.members` could be `Content::Members`... We should probably deserialize directly from the `element` object, or am I missing something?